pom.xml

```diff
@@ -6,7 +6,7 @@
     <groupId>com.linkedin.sparktfrecord</groupId>
     <artifactId>spark-tfrecord_2.11</artifactId>
     <packaging>jar</packaging>
-    <version>0.1.1</version>
+    <version>0.1.2</version>
```

Comment:
This is a backward incompatible change (it does not work with spark 2.3 anymore). Please bump the minor version. Use 0.2.0 instead.
After this, we should be able to merge it.